govcd/vm_dhcp_test.go

```diff
@@ -0,0 +1,228 @@
+// +build nsxv vm functional ALL
+
+/*
+ * Copyright 2019 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+	. "gopkg.in/check.v1"
+)
+
+// Test_VMGetDhcpAddress proves that it is possible to wait until DHCP lease is acquired by VM and
+// report the IP even if VM does not have guest tools installed.
+// The test does below actions:
+// 1. Creates VM
+// 2. Ensures there is a DHCP configuration for network
+// 3. Backs up network configuration for VM
```

Comment:
"Backs up" sounds as if you're saving network configuration for reverting it back in the future.

Reply:
Indeed. It used to do so before, but I was not able to reuse the existing SetupSuite VM so there is no need to backup/restore network config.
Changed